pom.xml

```diff
@@ -84,64 +84,66 @@
         <hbase-1.version>1.4.14</hbase-1.version>
         <kafka.version>2.8.2</kafka.version>
         <zkclient.version>0.11</zkclient.version>
-        <postgres.version>42.6.0</postgres.version>
+        <postgres.version>42.7.0</postgres.version>
 
         <curator.version>4.3.0</curator.version>
-        <netty.version>4.1.100.Final</netty.version>
+        <netty.version>4.1.101.Final</netty.version>
         <jetty.version>9.4.53.v20231009</jetty.version>
         <fasterxml.jackson.version>2.15.3</fasterxml.jackson.version>
         <fasterxml.woodstox.version>5.4.0</fasterxml.woodstox.version> <!-- managed from 5.3.0 -->
         <commons.net.version>3.10.0</commons.net.version> <!-- managed from 3.1 -->
-        <commons.text.version>1.10.0</commons.text.version>
+        <commons.text.version>1.10.0</commons.text.version> <!-- NOTE: versions >1.10.x break the accumulo tests -->
```

Comment:
is this comment indicating that 1.10.1+ doesn't work, but presumably 1.10.0 does?

Reply:
oh, i missed the `>` sign...

```suggestion
        <commons.text.version>1.10.0</commons.text.version> <!-- NOTE: versions 1.11+ break the accumulo tests -->
```